### PR TITLE
expand READMEs for Zapi and ZapiPerf

### PR DESCRIPTION
**Pull request by vgratian**
added READMEs for the two collectors:
- ZapiPerf
- Zapi

this is just as starting point, need to add more info and polish later on.

### CODE REVIEW

**Review by rahulguptajss — DISMISSED** [review]

File: cmd/collectors/zapi/README.md [hunk 1]
```diff
@@ -1,167 +1,50 @@
...
 
-<center><img src="examples/ontap_user_sm_2.png" width="75%"></center>
+The Zapi collector does not have the parameters `instance_key` and `override` parameters.
 
-In the left column, under *Users*, click on *Add* to create a new user. Choose a username. Under *Role*, select the role that we just created. Under *User Login Methods* select *ONTAPI*, and select one of the two authentication methods. Type in a password if you chose *Password*. Click on *Save*
-<center><img src="examples/ontap_user_sm_3.png" width="75%"></center>
+#### `counters`
 
-If you chose *Password*, you can add the username and password to the Harvest configuration file and start Harvest. If you chose *Certificate* jump to [Using Certificate Authentication](#using-certificate-authentication) to generate certificates files.
+This section should contain the complete or partial attribute tree of the API that is queried. Since the collector does get counter metadata from the ONTAP system, two additional symbols are used for non-numeric attributes:
```
Comment (cgrinds) [inline cmd/collectors/zapi/README.md:31]: This section contains the complete or partial attribute tree of the queried API.

**Review by cgrinds — APPROVED** [review]
Thanks @vgratian - made some minor edits. Nice job, these help.
Once you commit I'm going to update the examples

**Comment by vgratian** [comment]
@cgrinds, thanks for the comments, I'll update soon

**Comment by vgratian** [comment]
> Thanks @vgratian - made some minor edits. Nice job, these help.
> Once you commit I'm going to update the examples

@cgrinds, I'm done with the commit!

**Review by cgrinds — APPROVED** [review]

**Pull request merged**